test/compilable/cppmangle3.d

```diff
@@ -0,0 +1,25 @@
+
+
+extern(C++, "true")
+{
+}
+
+extern(C++, "__traits")
+{
+}
+
+extern(C++, "foo")
+{
+}
+
+int foo; // no name clashing with above namespace
+
+extern(C++, "std.chrono")
```

Comment:
I reckon '::' should probably be the separator in this case, since it specifically describes a C++ namespace, and is not necessarily valid D identifiers (as your example), so it probably shouldn't look like a D identifer.

Reply:
Or it can be separate arguments, then we don't have to worry about which separator to use or to parse a string, i.e. `extern (C++, "std", "chrono")`.

Reply:
I'm with Jacob.